book/version_control.md

```diff
@@ -53,9 +65,11 @@ You should **not** include the following in your code repository:
 
 See [](excluding-from-git) for details on how to mitigate the risk of including sensitive information in a Git repository.
 
-You might also want to include example [configuration](configuration.md) files, to help users trying to run your code understand what parameters need to be set. However, note that sometimes configuration files contain sensitive data or credentials, which should not be commited to version control. In this case, example configuration files should contain clear dummy values that are indicative of the expected values to be filled in.
+It is again worth stressing the importance of not committing sensitive, unpublished or disclosive data to your Git history. If you would like to include an example for end-users, a minimal dummy dataset can be committed to the repository. However, dummy datasets following the same layout can disclose the types of variables in the real data. As such, you should consider the inclusion of this data with caution, particularly for repositories intended to be hosted publicly.
```

Comment:
I wonder if this might be too off-putting for people considering publishing dummy data. There are many cases where doing so is really not an issue at all, and it adds a lot of value for users, so perhaps we need to explain this more clearly.

Reply:
If it motivates people to get someone to review the dummy data they are considering committing, or to document its creation, then to me it seems useful.

Reply:
My concern is that this is something people are already not doing enough. Perhaps we can be more specific about what a public dummy dataset should and shouldn't include. Currently this implies that we shouldn't publish any code referring to real variable names and types, which is very restrictive.

Reply:
Agreed, "types of variables" is quite a dampener and begs an example or link.